src/examples/src/config.tsx

```diff
@@ -341,6 +342,15 @@ export const config = {
 				}
 			}
 		},
+		form: {
+			filename: 'index',
+			overview: {
+				example: {
+					filename: 'Basic',
```

Comment:
I think that this could do with having more examples created, with the `Basic` example being very basic. I will raise an issue for you.